nixos/modules/services/web-apps/zitadel.nix

```
@@ -0,0 +1,223 @@
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.services.zitadel;
+
+  settingsFormat = pkgs.formats.yaml { };
+in {
+  options.services.zitadel =
+    let inherit (lib) mdDoc mkEnableOption mkOption mkPackageOptionMD types;
+    in {
+      enable = mkEnableOption
+        (mdDoc "ZITADEL, a user and identity access management platform.");
```

I believe the `mdDoc` is no longer necessary.  Based on my cursory understanding it was only added as a transitional function and now it's just a synonym for `id`.

```suggestion
    let inherit (lib) mkEnableOption mkOption mkPackageOptionMD types;
    in {
      enable = mkEnableOption
        "ZITADEL, a user and identity access management platform.";
```

Same comment for all of the other `md` utilities (like `mkPackageOptionMD`, which you can now replace with `mkPackageOption`)